locale/fi/LC_MESSAGES/django.po

```diff
@@ -2842,6 +2858,14 @@ msgstr "Poista käyttäjämerkintä"
 msgid "REMOVE_USER_TAG_CONFIRMATION"
 msgstr "Vahvista käyttäjämerkinnän poisto"
 
```

Comment:
The finnish translation file is missing `REMOVE_SUBMISSION_TAG`, making it fall back to english

![image](https://github.com/apluslms/a-plus/assets/3648442/fe6fd0c0-3d96-4dcd-8f74-87b938b12a83)

Reply:
Added